src/io/inputs/model-converter/convertorVisitor.cpp

```diff
@@ -162,10 +216,12 @@ std::any ConvertorVisitor::visitIdentifier(ExprParser::IdentifierContext* contex
 
 std::any ConvertorVisitor::visitMuldiv(ExprParser::MuldivContext* context)
 {
+    std::string op = context->op->getText();
+    op == "*" ? parentsStack_.push_back({"multiplication", typeid(MultiplicationNode)})
+              : parentsStack_.push_back({"division", typeid(MultiplicationNode)});
```

Comment:
Avoid temporary objects with `emplace_back`

```suggestion
    op == "*" ? parentsStack_.emplace_back("multiplication", typeid(MultiplicationNode))
              : parentsStack_.emplace_back("division", typeid(MultiplicationNode));
```